### DISCUSSION TITLE
#30. VATZ biweekly meeting at 2023-03-29 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#416] [General]
### 1. Overall
<img width="1208" alt="image" src="https://user-images.githubusercontent.com/6308023/225478550-7b20d967-b0d8-475a-b86f-4678e8697baa.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
28 | 6/25 |  24
29 | 6/23 |  26

---

Current Traffic

<img width="922" alt="image" src="https://user-images.githubusercontent.com/6308023/228414149-bfdc9c0d-f58b-4f34-9600-fd2fd413d5bb.png">

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/422
- https://github.com/dsrvlabs/vatz/issues/323
- https://github.com/dsrvlabs/vatz/issues/388
- https://github.com/dsrvlabs/vatz/issues/415

# External user accessibility 
- https://github.com/dsrvlabs/vatz/issues/395
   - https://github.com/dsrvlabs/vatz/issues/397
   - https://github.com/dsrvlabs/vatz/issues/404 

# co-working issue
- https://github.com/dsrvlabs/vatz/issues/87
- https://github.com/dsrvlabs/vatz/issues/401
   - https://github.com/dsrvlabs/vatz/issues/302
   - https://github.com/dsrvlabs/certik-project/issues/137 
- https://github.com/dsrvlabs/vatz/issues/421

**Comment by rootwarp** [comment]
# 준교

- https://github.com/dsrvlabs/vatz/issues/419
- https://github.com/dsrvlabs/vatz/issues/421

**Comment by gnongs** [comment]
## 진홍
- no issue

**Comment by kim201212** [comment]
# 형기

- No issue

**Comment by heejin-github** [comment]
## 희진
- https://github.com/dsrvlabs/vatz/issues/401
- https://github.com/dsrvlabs/vatz/issues/418
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/30
- https://github.com/dsrvlabs/vatz/issues/423

**Comment by skonhwang** [comment]
## 승곤
- no issue

**Comment by qkrybin** [comment]
## 영빈
- no issue

**Comment by meetrick** [comment]
## 황제
- no issue
